packages/test-utils/src/wrapper.js

```diff
@@ -716,7 +716,9 @@ export default class Wrapper implements BaseWrapper {
           const isUpdated = Object.keys(data).some(key => {
             return (
               // $FlowIgnore : Problem with possibly null this.vm
-              this.vm[key] === data[key] || this.vm.$attrs[key] === data[key]
+              this.vm[key] === data[key] ||
+              // $FlowIgnore : Problem with possibly null this.vm
+              (this.vm.$attrs && this.vm.$attrs[key] === data[key])
```

Comment:
I had a few issues in CI recently in regards to this guy in vue 2.3. Looks like I introduced a small regression in VTU for `setProps` for vue version 2.3 in #1618. The stack trace was a bit difficult to track down, but this should fix the issues seen